assets/css/_sass/website/_utilities.scss

```diff
@@ -38,3 +38,15 @@
     top: 25%;
   }
 }
+
+/* Text align classes */
+
+.align-left {
+  text-align: left !important;
+}
+.align-right {
+  text-align: right !important;
+}
+.align-center {
```

Comment:
FYI: `helix-ui` provides `.hxLeft`, `.hxCenter`, and `.hxRight` helpers, but it doesn't `!important`. Is the `!important` absolutely necessary?